src/shared/components/query/QueryAndDownloadTabs.tsx

```diff
@@ -37,7 +38,7 @@ export default class QueryAndDownloadTabs extends React.Component<IQueryAndDownl
 	constructor(props:IQueryAndDownloadTabsProps){
 		super();
 
-		if (getBrowserWindow().localStorage.getItem(QUICK_SEARCH_LS_KEY) === QUICK_SEARCH_TAB_ID) {
+		if (getBrowserWindow().localStorage.getItem(QUICK_SEARCH_LS_KEY) === QUICK_SEARCH_TAB_ID && props.showQuickSearchTab) {
```

Comment:
@dippindots minor thing - i prefer putting the props condition first. When i look at the file i like to mentally be able to blank out certain parts of the code based on the configuration of the component, so i know what i can ignore when im for instance not working on quick search. In this case it might also be slightly faster (although prolly negligible), because you don't have to do a localStorage lookup when `showQuickSearchTab ` is false. With `&&` the second condition does not get executed if the first is false

Reply:
Thanks Ino, I just saw this. Updated.